src/consensus/params.h

```diff
@@ -48,6 +48,7 @@ struct Params {
     int nBudgetProposalEstablishingTime; // in seconds
     int nSuperblockStartBlock;
     int nSuperblockCycle; // in blocks
+    int nMinGovernanceQuorum; // Min absolute vote count to trigger an action 
```

Comment:
I debated calling it nGovernanceMinQuorum but opted for this order because when expanded "minimum governance quorum" sounds better than "governance minimum quorum".

Also I think min is a very widely understood abbreviation for minimum, especially among programmers.